docs/maintainers/patch-release.md

```diff
@@ -0,0 +1,44 @@
+# Creating a patch release
+
+## Overview
+Occasionally, it can be necessary to create a patch release of `play-frontend-hmrc` from an older (i.e. not lastest) 
+version of the library. For example, it might be necessary to bump an older version of `hmrc-frontend` or `govuk-frontend`.
+
+An example of when this was necessary was when the Tudor Crown logo was added to `govuk-frontend`. A tight deadline was 
+mandated by Government Digital Service (GDS), shortly after they had released a breaking major release (see notes from 
+GDS [here](https://us1.campaign-archive.com/?e=__test_email__&u=ada1bae4b177beb9e03502ced&id=2325298b37&utm_source=GOV.%E2%80%8CUK+Design+System&utm_campaign=aabdb157a5-FRONTEND_CROWN_05022024&utm_medium=email&utm_term=0_367f0a5723-aabdb157a5-536304558])).
+
+## Approach
+
+The main concept is to create a branch from an older tag of `play-frontend-hmrc`, then create a "hotfix" release from 
```

Comment:
I like the explanations here, and will probably steal some of this for the docs in hmrc-frontend 😅